src/App/Services/MobilePlatformUtilsService.cs

```diff
@@ -218,7 +218,7 @@ public async Task<bool> SupportsBiometricAsync()
                         var supportsFace = await _deviceActionService.SupportsFaceBiometricAsync();
                         text = supportsFace ? AppResources.FaceIDDirection : AppResources.FingerprintDirection;
                     }
-                    var fingerprintRequest = new AuthenticationRequestConfiguration(text)
+                    var fingerprintRequest = new AuthenticationRequestConfiguration(text, text)
```

Comment:
What's the second param for now?

Reply:
The params are now `title` and `reason` (was originally only `region`).  I'm not sure why they did this because from what I can tell one is used with Android and the other with iOS, yet both serve the same purpose.